README.md

```diff
@@ -769,6 +770,57 @@ loop = asyncio.get_event_loop()
 loop.run_until_complete(main())
 ```
 
+## Logging
+
+> Feature appears in version 1.5.0
+
+SDK v1.5.0 introduces logging for debug purposes.
+Logs are disabled by default, thus SDK behavior remains the same. Logging should be enabled explicitly via client configuration:
+
+```py
+from okta.client import Client as OktaClient
+
+
+config = {"logging": {"enabled": True}}
+client = OktaClient(config)
+```
+
+SDK utilizes standard python library `logging`. By default, log level INFO is set. You can set another log level via config:
+
+```py
+from okta.client import Client as OktaClient
+import logging
+
+config = {"logging": {"enabled": True, "logLevel": logging.DEBUG}}
+client = OktaClient(config)
+```
+
+**NOTE**: DO NOT SET DEBUG LEVEL IN PRODUCTION!
+
+Here's a complete example:
+```python
+from okta.client import Client as OktaClient
+import asyncio
+import logging
+
+async def main():
+    config = {"logging": {"enabled": True, "logLevel": logging.DEBUG}}
+    client = OktaClient(config)
+    users, resp, err = await client.list_users()
+    assert users is not None
+
+loop = asyncio.get_event_loop()
+loop.run_until_complete(main())
+```
+
+You should see in logs something like the following (some values redacted only for readme and willbe real in your case, thus do not use DEBUG level in production):
```

Comment:
Change To: "You should now see logs in your console. Actual API Tokens will be logged to the console, so use caution and never use `DEBUG` level logging in production.

Reply:
Fixed